.github/ISSUE_TEMPLATE/security_bug_report.md

```diff
@@ -0,0 +1,8 @@
+---
+name: 🔥 Security bug report
+about: How to report security vulnerabilities
+---
+
+For all security related bugs, email security@ansible.com instead of using this issue tracker and you will receive a prompt response.
```

Comment:
@gundalow how about `security+molecule@ansible.com`? Anybody practices email tags?

Reply:
Interesting idea, though this would be zero to low traffic and possibly break other peoples mail filters.